drivers/video/gc2145.c

```diff
@@ -0,0 +1,1148 @@
+/*
+ * Copyright (c) 2024 Felipe Neves
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT gc_gc2145
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+
+#include <zephyr/drivers/video.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/gpio.h>
+
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(gc2145, CONFIG_LOG_DEFAULT_LEVEL);
```

Comment:
There is now a `CONFIG_VIDEO_LOG_LEVEL` which you can use instead :)
- https://github.com/zephyrproject-rtos/zephyr/pull/76393